public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -58,20 +59,38 @@ const LogMessage = ({ hasAnsi, entry, highlights, styles }: LogMessageProps) =>
   return <>{entry}</>;
 };
 
-const restructureLog = (line: string, prettifyLogMessage: boolean): string => {
+function extractFirstLine(input: string): string {
+  // Regular expression to match everything up to the first occurrence of '\r\n'
+  const regex = /^(.*?)(\r\n|$)/;
```

Comment:
```suggestion
  const regex = /^(.*?)(\r\n|\n|$)/;
```
CRLF line endings are primarily used in Windows operating systems; Unix-based systems like Linux and macOS typically use LF for line endings.